### PR TITLE
feat: added events page

### PR DESCRIPTION
**Pull request by indraasura**
Created page for events

### CODE REVIEW

File: events.html [hunk 1]
```diff
@@ -0,0 +1,338 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>DSC ATC | Team Members</title>
```
Comment (shashank-sharma) [inline events.html]: Change Title

File: events.html [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.4.1/jquery.min.js"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" type="text/css" href="css/main.css">
+    <link rel="stylesheet" type="text/css" href="css/tilter.css">
```
Comment (shashank-sharma) [inline events.html]: tilter.css not required including anime.js

File: events.html [hunk 3]
```diff
@@ -0,0 +1,338 @@
...
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/animejs/2.2.0/anime.min.js"></script>
+    <script src="js/main.js"></script>
+    <script src="js/tilter.js"></script>
```
Comment (shashank-sharma) [inline events.html]: Not required

File: events.html [hunk 4]
```diff
@@ -0,0 +1,338 @@
...
+                  <p>Had a great time organising DSC ATC’s first workshop on Git and Github, 
+                      where we provided a hands-on experience and touched several topics about Github platform 
+                      and version-control tracking Git.</p>
+                </div>
+                <div class="card-action">
+                  <button class="btn waves-effect green darken-3"><a href="#" class="white-text">Visit Event Page</a></button>
+                </div>
+              </div>
+        </div>
+        <div class = "col s12 m4 l4">
```
Comment (shashank-sharma) [inline events.html]: Can you remove duplicate cards, or add coming soon card with workable button

File: index.html [hunk 5]
```diff
@@ -18,12 +18,12 @@
 	<meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1"/>
 </head>
 <body>
-<div class="bouncing-loader" id="preloader">
+<!-- <div class="bouncing-loader" id="preloader">
```
Comment (shashank-sharma) [inline index.html]: Remove comment for loader

**Review by shashank-sharma — CHANGES_REQUESTED** [review]
Before merging it and showing it in production site, please change :)

File: events.html (hunk 1 above)
Comment (indraasura) [reply events.html]: Cool, done!

File: events.html (hunk 4 above)
Comment (indraasura) [reply events.html]: okay, will do it

File: events.html (hunk 2 above)
Comment (indraasura) [reply events.html]: done

**Pull request closed without merging**